src/theme/index.ts

```diff
@@ -9,8 +8,11 @@ import { useTheme } from "styled-components";
 export const themes: Record<ThemeName, Theme> = {
   dark: merge({}, theme, darkTheme),
   light: merge({}, theme, lightTheme),
-  classic: merge({}, theme, classicTheme),
 };
+
+/**
+ * @deprecated The 'classic' theme has been removed and will fallback to 'light'. Please use 'light' or 'dark' instead.
```

Comment:
this deprectaes the entire type, which is not exactly correct
I don't think there's a way to use the `@deprectaed` tag on one of the union members. We can just leave a regular comment, perhaps to the "classic" option specifically.